### PR TITLE
Aqua-park task complete

### PR DESCRIPTION
**Pull request by thorium-zero**
Yeah, I know about me using  that `Exception` class is kinda weird and stuff, but there are no instructions on how to handle exceptions for this project, so... :)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,25 +1,74 @@
...
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        super().__init__()
```
Comment (Oleg-s-s) [inline app/main.py]: Why you need this?

File: app/main.py [hunk 2]
```diff
@@ -1,25 +1,74 @@
...
+    def __init__(self, name: str,
+                 limitation_class: SlideLimitationValidator) -> None:
+        self.name: str = name
+        self.limitation_class: SlideLimitationValidator = limitation_class
+
+    def can_access(self, visitor: Visitor) -> bool:
+        result = True
+        try:
+            self.limitation_class(visitor.age, visitor.weight, visitor.height)
+        except Exception:
```
Comment (Oleg-s-s) [inline app/main.py]: I saw your comment but still using `Exception` directly can lead to missing some important ones. I think `ValueError` here will be enought

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (thorium-zero) [reply app/main.py]: Yeah, I know, will fix in the next commit

File: app/main.py [hunk 3]
```diff
@@ -1,25 +1,73 @@
...
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+    def is_valid(self) -> bool:
+        pass
```
Comment (YuriiKindrat) [inline app/main.py]: why do u need this ?)

File: app/main.py [hunk 4]
```diff
@@ -1,25 +1,73 @@
...
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(14, 60)
+    weight = IntegerRange(50, 120)
+    height = IntegerRange(120, 220)
+
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        super().__init__(age, weight, height)
```
Comment (YuriiKindrat) [inline app/main.py]: no need to rewrite init methods without changes

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 3 above)
Comment (thorium-zero) [reply app/main.py]: I don't, I though I will, though. And then I forgot to remove it

**Review by YuriiKindrat — APPROVED** [review]

**Review by TPolina — APPROVED** [review]

**Pull request closed without merging**